### PR TITLE
Remove erroneous MichelTestPrefs.

### PR DESCRIPTION
**Pull request by bbastings**

### CODE REVIEW

**Review by pmconne — APPROVED** [review]
Can somebody remind me why we have code in the monorepo that is not compiled during CI? @NancyMcCallB

File: test-apps/ui-test-app/src/frontend/index.tsx [hunk 1]
```diff
@@ -334,7 +334,6 @@ export class SampleAppIModelApp {
 
     await FrontendDevTools.initialize();
     await HyperModeling.initialize();
-    await MapLayersUI.initialize({ iTwinConfig: new MichelTestPrefs(), featureInfoOpts: { onMapHit: DefaultMapFeatureInfoTool.onMapHit } });
```
Comment (raplemie) [inline test-apps/ui-test-app/src/frontend/index.tsx:337]: @mdastous-bentley Should the full MapLayersUI be removed, or only the iTwinConfig ?

**Comment by raplemie** [comment]
> Can somebody remind me why we have code in the monorepo that is not compiled during CI? @NancyMcCallB

These were removed because of build time complaints and that the applications were not deemed easy to fix when broken by something else. Not sure where this complain came from though.

**Review by raplemie — APPROVED** [review]
LGTM

**Pull request merged**